server.js

```diff
@@ -27,7 +27,7 @@
 ///////////////////////////////////////////////////
 
 var env = require('./env')( );
-
+var translate = require('./lib/language')(env.defaults.language).translate;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['translate' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/797646480/issues/source?bid=2452379&fileBranchId=2723722#l30)